sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/CheckpointStore/BlobsCheckpointStore.cs

```diff
@@ -23,8 +21,10 @@ namespace Azure.Messaging.EventHubs.Processor
     ///   A storage blob service that keeps track of checkpoints and ownership.
     /// </summary>
     ///
-    internal sealed class BlobsCheckpointStore : StorageManager
+    internal sealed partial class BlobsCheckpointStore : StorageManager
     {
+        private const string BlobsResourceDoesNotExist = "The Azure Storage Blobs container or blob used by the Event Processor Client does not exist.";
```

Comment:
nit:  Please add doc comments.

Reply:
This breaks the pattern of using resources for user-facing text.   Should we consider making this a partial so that the bindings can use a different form rather than having a single deviation?

Reply:
Let me see what I can do here.

Reply:
I appreciate it.   Definitely non-blocking, if it turns out to be infeasible or high-effort.